<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>production_feature_window</class>
 <widget class="QDialog" name="production_feature_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>525</width>
    <height>377</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>型号特征</string>
  </property>
  <property name="windowIcon">
   <iconset theme="产品型号" resource="resources.qrc">
    <normaloff>:/resourse/canshu.png</normaloff>:/resourse/canshu.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>屏幕区域：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>产品型号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>方案配置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>54</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>外形尺寸：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>分 辨 率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(85, 255, 127);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>319</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="autoRepeatDelay">
    <number>295</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>屏幕类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>屏幕尺寸：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>209</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>刷屏方式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);border-color: rgb(85, 255, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
